src/main/webapp/script.js

```diff
@@ -12,6 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ * Function that will call other functions when the page loads. 
+ */
+function loadIndex() {
+  addAutoResize();
+  fetchAuthentication();
+  fetchForum();
+  loadNotifications();
```

Comment:
What if we rename this function to fetchNotifications()? Because it functions in a similar way to the other two fetch functions. (You would also need to reorder the function to where fetch functions are located.

Reply:
done